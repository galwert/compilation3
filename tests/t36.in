int func() {
    return 1;
}

void main(){
    byte small = 255 b;
    func = 5;
}