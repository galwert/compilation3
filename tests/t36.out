---end scope---
line 7: variable func is not defined
